test/integration/targets/docker_container/tasks/tests/image-ids.yml

```diff
@@ -63,6 +63,28 @@
     force_kill: yes
   register: create_4
 
+- name: Untag image
+  # Image will not be deleted since the container still uses it
+  docker_image:
+    name: alpine:3.8
+    force_absent: yes
+    state: absent
+
+- name: Create container with alpine image via name (check mode, will pull, same image)
```

Comment:
I would do this test one more time after create_6 to be sure that changed will be false. And also create_6 after that again (so twice in actual mode for one image), just in case, and also check that changed will be false.

Reply:
That's not necessary, idempotence for started and created containers is already tested in `start-stop.yml`.